lutris/gui/views/list.py

```diff
@@ -116,13 +118,13 @@ def set_sort_with_column(self, col, sort_col):
         self.model.set_sort_func(col, sort_func, sort_col)
 
     def get_selected_item(self):
```

Comment:
Isn't it get_selected_items() now? If not, what happens when multiple items are selected?

Reply:
Modified already existing get_selected_items() to do the job. Will rename it to item

Reply:
Well, list-returning methods should be plural to make it clearer that they return a list. But you've done that; I think there's one stray 'selected_item' variable, no big deal.